functorch/_src/decompositions.py

```diff
@@ -277,3 +277,35 @@ def _s_where_canonicalization(a, b, c):
 @register_decomposition(aten.detach)
 def detach_decomposition(x):
     return x
+
+
+@register_decomposition(aten.var)
+def var_decomposition(x, dims, correction=0, keepdim=False):
+    if dims is None:
```

Comment:
Same for this.

Reply:
So, this can happen because of this

https://github.com/pytorch/pytorch/blob/64670e414eb4bcba7dcc382bb16be1492cdd6e95/aten/src/ATen/native/native_functions.yaml#L4704-L4708

where dim arg is Optional and can be None.

But, when we later call mean and use dim, it can't handle NoneType 

https://github.com/pytorch/pytorch/blob/64670e414eb4bcba7dcc382bb16be1492cdd6e95/aten/src/ATen/native/native_functions.yaml#L2985-L2990